src/blueprint/from_config/operators/ref_field.rs

```diff
@@ -0,0 +1,35 @@
+use crate::blueprint::*;
+use crate::config;
+use crate::config::{Config, Field};
+use crate::lambda::Expression;
+use crate::try_fold::TryFold;
+use crate::valid::Valid;
+
+pub fn update_ref_field<'a>(
+  is_add_field: bool,
+) -> TryFold<'a, (&'a Config, &'a Field, &'a config::Type, &'a str), FieldDefinition, String> {
+  TryFold::<(&Config, &Field, &config::Type, &str), FieldDefinition, String>::new(
+    move |(config, field, _, _), b_field| {
+      if is_add_field {
+        return Valid::succeed(b_field);
+      }
+      if !field.has_resolver() {
+        let field_type = config.types.get(&field.type_of);
+        if let Some(ty) = field_type {
+          let mut mut_field = b_field;
+          let has_resolver = ty.fields.iter().any(|f| f.1.has_resolver());
```

Comment:
I think here could be a problem since we are not checking fields recursively here. Can we add another test for resolving value with more deep nesting? similar to the `tests/graphql/passed/test-ref-other.graphql` to verify it?

Something like this:
```
type User {
  name: String @http(path: "/users/1")
}

type User1 {
  user1: User
}

type Query {
  firstUser: User1
}
```

Reply:
right, thanks